skimage/feature/masked_register_translation.py

```diff
@@ -0,0 +1,275 @@
+"""
+Implementation of the masked normalized cross-correlation (MNXC)
+
+Based on the following publication:
+D. Padfield. Masked object registration in the Fourier domain.
+IEEE Transactions on Image Processing (2012)
+
+and the author's original MATLAB implementation, available on this website:
+http://www.dirkpadfield.com/
+"""
+
+import numpy as np
+from functools import partial
+
+# next_fast_len was implemented in scipy 0.18
+# In case it cannot be imported, we use the id function

```

Comment:
Please, add a note in the TODO.txt file under the section "Others" asking to remove this piece of code when the min dep. version of scipy is set to 0.18.